INSTALL.md

```diff
@@ -11,11 +11,12 @@ $ sudo yum groupinstall "Development Tools"
 
 
 ## To compile and install Singularity from a released tarball:
-Assuming a 2.3 released tarball...
+Assuming a 2.3.1 released tarball...
 ```
-$ wget "https://github.com/singularityware/singularity/releases/download/2.3/singularity-2.3.tar.gz"
-$ tar -xvzf singularity-2.3.tar.gz
-$ cd singularity-2.3
+$ version=2.3.1
+$ wget "https://github.com/singularityware/singularity/releases/download/${version}/singularity-${version}.tar.gz"
+$ tar -xvzf singularity-${version}.tar.gz
+$ cd singularity-${version}
 $ ./configure --prefix=/path/to/singularity
 $ make
```

Comment:
one more quick comment here! I think given that most don't know the linux filesystem hierarchy, we should give an example like `prefix=/usr/local` so we don't see people installing it in home, Downloads, etc.